Documentation/SDK_Tipping_Support_Documentation.md

```diff
@@ -72,7 +72,7 @@ The tipping style is pre-defined per market and is not configurable.
 | Germany | Total Amount |
 
 ### Is there a MAX level for tip?
-To prevent accidental or invalid amounts (for example if the customer enters their PIN code instead of a tip amount), the maximum tip level is set to 50%. If the customer enters more than 50%, the card reader will show a “too high amount” warning message.
+To prevent accidental or invalid amounts (for example if the customer enters their PIN code instead of a tip amount), there is a maximum tip level. The limit for Europe is 50% and for the US the limit is 100%. If the customer enters more than the limit, the card reader will show a “too high amount” warning message.
```

Comment:
There is a maximum level validation to prevent accidental or invalid entering of amounts. This can happen for example if customers enter their PIN code instead of a tip amount. The maximum limit for Europe is 50%, and 100% for the US. If this level is exceeded, the card reader will display a “Too high amount” warning message.